### PR TITLE
feature(ui-ux): `translations` - add option for es and it translations

### LINKED ISSUE #2696: `translations` - add option for `es` and `it` translations
#### What would you like to be added:
Linked https://github.com/DeFiCh/wallet/pull/2688

- [ ] Add option on language list in settings page
- [ ] Display translated labels

### PR DESCRIPTION
**Pull request by chloezxyy**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- [x]  Add option on language list in settings page
- [x] Display translated labels

#### Which issue(s) does this PR fixes?:

Fixes #2696

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

**Comment by pierregee** [comment]
Is there a shorter text for `SWAP` in Español?, it seems to be cut right now. Or else we might need handle it in the UI

Android: image

File: shared/translations/languages/es.json [hunk 1]
```diff
@@ -0,0 +1,1387 @@
+{
+    "screens/WalletNavigator": {
+        "Guidelines": "Instrucciones",
```
Comment (fullstackninja864) [inline shared/translations/languages/es.json:3]: use 2 space indentation

File: shared/translations/languages/es.json [hunk 2]
```diff
@@ -0,0 +1,1387 @@
...
+    },
+    "components/Announcements": {
+        "DETAILS": "Ver"
+    },
+    "components/PricesSection": {
+        "PRICES": "PRECIOS",
+        "{{tokenA}} price in {{tokenB}}": "{{tokenA}} precio en {{tokenB}}",
+        "{{tokenB}} price in {{tokenA}}": "{{tokenB}} precio en {{tokenA}}"
+    }
+}
```
Comment (fullstackninja864) [inline shared/translations/languages/es.json:1387]: add new line at end

File: shared/translations/languages/it.json [hunk 3]
```diff
@@ -0,0 +1,1381 @@
+{
+    "screens/WalletNavigator": {
+        "Guidelines": "Linee guida",
```
Comment (fullstackninja864) [inline shared/translations/languages/it.json:3]: use 2 space indentation

File: shared/translations/languages/it.json [hunk 4]
```diff
@@ -0,0 +1,1381 @@
...
+    },
+    "components/Announcements": {
+        "DETAILS": "DETTAGLI"
+    },
+    "components/PricesSection": {
+        "PRICES": "PREZZI",
+        "{{tokenA}} price in {{tokenB}}": "Prezzo {{tokenA}} in {{tokenB}}",
+        "{{tokenB}} price in {{tokenA}}": "Prezzo {{tokenB}} in {{tokenA}}"
+    }
+}
```
Comment (fullstackninja864) [inline shared/translations/languages/it.json:1381]: new line at end

**Comment by fullstackninja864** [comment]
Can we recheck these type of hidden text issue on all screen please

**Review by fullstackninja864 — CHANGES_REQUESTED** [review]

**Comment by chloezxyy** [comment]
Modified :)

image

**Comment by chloezxyy** [comment]
Updated translations for shorter text mainly for for Spanish and title of DEX page for Italian
Screenshot 2022-04-28 at 5 17 08 PM

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**